js/vimlparser.js

```diff
@@ -2987,9 +2990,9 @@ ExprParser.prototype.parse_expr8 = function() {
                     if (token.type != TOKEN_SQCLOSE) {
                         throw Err(viml_printf("unexpected token: %s", token.value), token.pos);
                     }
+                    var left = node;
```

Comment:
yes? Maybe I just don't understand what you asked?

It's exactly translated as I expected.